modules/hostap/Kconfig

```diff
@@ -249,6 +249,9 @@ if WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE
 		default 16384
 endif
 
+config WIFI_NM_WPA_SUPPLICANT_ROAMING
+	bool "Roaming support"
```

Comment:
Could there be a help text also describing the supported roaming functionality

Reply:
Added help text.

Reply:
needs an elaborate text to describe the `roaming`.

Reply:
Updated.